src/main/kotlin/no/skatteetaten/aurora/cantus/controller/NexusController.kt

```diff
@@ -0,0 +1,36 @@
+package no.skatteetaten.aurora.cantus.controller
+
+import no.skatteetaten.aurora.cantus.service.NexusClient
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.RestController
+import reactor.core.publisher.Flux
+import reactor.core.publisher.Mono
+
+data class Version(
+    val name: String,
+    val lastModified: String
+)
+
+@RestController
+class NexusController(val nexusClient: NexusClient) {
+
+    @GetMapping("/versions")
+    fun getVersions(
+        @RequestParam namespace: String,
```

Comment:
Er det bevisst at namespace er brukt her? 

Er det bedre med imageGroup og imageName for å lettere se hva det handler om.
Kanskje skadet av å jobbe med openshift og kubernetes, men jeg tenker på namespace i kubernetes når jeg ser dette.

Reply:
Helt enig i at vi bør unngå begrepsforveksling her. `imageGroup` er et godt forslag; jeg tror jeg skal gå for det.